Maple2.Server.World/Service/WorldService.Chat.cs

```diff
@@ -60,4 +60,26 @@ public partial class WorldService {
             return Task.FromResult(new ChatResponse());
         }
     }
+    
+    private Task<ChatResponse> WorldChat(ChatRequest request) {
+        foreach ((int, ChannelClient) channel in channelClients) {
+            try {
+                channel.Item2.Chat(request);
+            } catch (RpcException ex) when (ex.StatusCode is StatusCode.NotFound) {
```

Comment:
Here and below, you shouldn't be checking for `NotFound` here, unlike `WhisperChat`, there are no expected bad statuses from this call. (Just remove this whole exception catching block)